pkgs/by-name/ev/evebox/package.nix

```
@@ -0,0 +1,32 @@
+{ lib
+, rustPlatform
+, fetchFromGitHub
+}:
+
+rustPlatform.buildRustPackage rec {
+  pname = "evebox";
+  version = "0.17.2";
+
+  src = fetchFromGitHub {
+    owner = "jasonish";
+    repo = "evebox";
+    rev = version;
+    hash = "sha256-djL5cdudJNPAWLMQPS2Dkcc9H/gouOuu8evcBDdY9wA=";
+  };
+
+  cargoLock = {
+    lockFile = "${src}/Cargo.lock";
```

```suggestion
    lockFile = ./Cargo.lock;
```

This is IFD, which is disallowed in nixpkgs. Please vendor the lock file

---

Funny, I already had it fixed before... (ofborg already informed me regarding this). I guess I force pushed from a wrong device two days ago... :joy:

---

`--force-with-lease` ;)

---

Oooh, that's brilliant, I didn't even know that option existed  :exploding_head: Thanks!